test/test_ops.py

```diff
@@ -647,6 +647,48 @@ def test_convert_boxes_to_roi_format(self):
                 self.assertTrue(torch.equal(ref_tensor, ops._utils.convert_boxes_to_roi_format(box_sequence)))
 
 
+class BoxTester(unittest.TestCase):
+    def test_bbox_xywh(self):
+        # Simple test convert boxes to xywh and back. Make sure they are same.
+        # box_tensor is in x1 y1 x2 y2 format.
+        box_tensor = torch.tensor([[0, 0, 100, 100], [0, 0, 0, 0],
+                                  [10, 15, 30, 35], [23, 35, 93, 95]], dtype=torch.float)
+        exp_xywh = torch.tensor([[0, 0, 100, 100], [0, 0, 0, 0],
+                                [10, 15, 20, 20], [23, 35, 70, 60]], dtype=torch.float)
+
+        box_xywh = ops.box_xyxy_to_xywh(box_tensor)
+        # print(box_xywh)
+        # print(box_xywh.shape)
+        assert(exp_xywh.size() == torch.Size([4, 4]))
+        assert(exp_xywh.dtype == box_tensor.dtype)
+        assert(torch.all(torch.eq(box_xywh, exp_xywh)).item() is True)
```

Comment:
For the purpose of learning:

1. `assert` is a [statement](https://docs.python.org/3/reference/simple_stmts.html#the-assert-statement) and not a function. Thus, the outer parentheses are not needed
2. The `assert expression` already tests if `bool(expression) is True`. Thus, the prefixed `is True` on a boolean is not needed.

You can apply this in multiple places below.

```suggestion
        assert exp_xywh.size() == torch.Size([4, 4])
        assert exp_xywh.dtype == box_tensor.dtype
        assert torch.all(torch.eq(box_xywh, exp_xywh)).item()
```

Reply:
Absolutely, I corrected it.